sublime_linter.py

```diff
@@ -169,6 +169,15 @@ def on_hover(self, view, point, hover_zone):
             if persist.settings.get('show_hover_region_report'):
                 SublimeLinter.shared_plugin().open_tooltip(view, point, True)
 
+    def on_selection_modified_async(self, view):
+        if not view:
+            return
+
+        lineno, colno = self.get_line_and_col(view)
```

Comment:
The `view` here doesn't have to be the `active_view`. 

- You can modify views in the background, and this will trigger.
- If you have multiple views into the same buffer, the view here will always be the *primary* one.

That's why I have the `State` object in `status_bar_view.py`, which holds the `active_view` and `current_pos`.